immobilus/logic.py

```diff
@@ -1,31 +1,59 @@
+"""Main library module to prepare the target `immobilus` class."""
+import calendar
+import copyreg
 import sys
 import time
-import calendar
 from asyncio import iscoroutinefunction
-from datetime import datetime, date, timedelta, tzinfo
+from datetime import date, datetime, timedelta, tzinfo
 from functools import wraps
 
 from dateutil import parser
 
-try:
-    import copy_reg as copyreg
-except ImportError:
-    import copyreg
 
-TIME_TO_FREEZE = None
-TZ_OFFSET = 0
+class Config(object):
+    """Changeable options."""
+
+    time_to_freeze = None
+    tz_offset = 0
 
 
 class UTC(tzinfo):
-    """UTC"""
+    """UTC."""
 
     def utcoffset(self, dt):
+        """Get zero timedelta.
+
+        Parameters:
+            dt: Any
+                not used at the moment
```

Comment:
It is not  "not used at the moment". This class repeats an interface of a real timezone instance, but returns fake data.